pyproject.toml

```diff
@@ -1,7 +1,8 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "setuptools",
+    # TODO: Remove the specifier once setuptools#2849 is resolved.
+    "setuptools<58.5",
```

Comment:
Maybe we can optimistically anticipate that a fix will be part of the next release?

```suggestion
    "setuptools!=58.5",
```

Reply:
Maybe the `!=` is not a good idea with `X.Y` version numbers when we expect `X.Y.Z` version numbers in the release. Ok with ignoring this suggestion to keep this fix simple.